.github/workflows/test-create-release.yml

```diff
@@ -46,7 +47,7 @@ jobs:
             const release_id = process.env.RELEASE_ID
             const { owner, repo } = context.repo
             try {
-              await github.repos.deleteRelease({ owner, repo, release_id })
+              await github.rest.repos.deleteRelease({ owner, repo, release_id })
```

Comment:
@quesabe I didn't know we can run JS right in the pipeline 😳

Reply:
Yeah. It's a special action that parses JS and runs it. Id does some basic stuff and exposes GitHub API. Good for simple manipulations.